server/svix-server/src/v1/endpoints/message.rs

```diff
@@ -225,6 +225,7 @@ async fn list_messages(
 pub struct CreateMessageQueryParams {
     #[serde(default = "default_true")]
     with_content: bool,
+    insert_event_type: Option<bool>,
```

Comment:
We need a better name probably. Dunno what it is though. :)

Reply:
No strong opinion on this one. `insert_event_type` is a bit of a mouthful but I think it's pretty clear what it does.

Reply:
`inject_event_type`?

Reply:
Renamed to `inject_event_type`.